locals.tf

```diff
@@ -29,4 +29,11 @@ locals {
     account.id
     if account.name == "Images (${local.sharedservices_account_type})"
   ][0]
+
+  # Turn the prefix list CIDRS for the S3 gateway endpoint into a list of OpenVPN
+  # friendly "network netmask" entries.
+  private_networks = [
```

Comment:
Do we want to call this local variable something different to avoid confusion with `var.private_networks`?  Maybe just `vpc_endpoint_cidrs`?

Reply:
That's a good suggestion.

Reply:
Since we deconstruct the CIDR into a network and netmask is there a more accurate description? My main issue is that since they are not in CIDR notation I do not want the variable name to cause a mistaken assumption.

Reply:
I think this question got lost in the tail end of Friday and I would appreciate input from @cisagov/team-ois.

Reply:
How about `private_networks_and_s3`?  I agree with @dav3r that it's quite easy to confuse `local.private_networks` with `var.private_networks`, and that's a good name to give the local a more specific name.

Reply:
Oh I definitely agree with changing the variable name. I just had issues with `vpc_endpoint_cidrs` as the replacement.

Reply:
How about just `vpc_endpoints` then?

Reply:
Updated in 25aa490.